packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/SfoxCheckout/Content/OrderCheckout.js

```diff
@@ -64,17 +58,19 @@ const OrderCheckout = ({ quoteR, account, onFetchQuote, reason, finishAccountSet
           <Text size='14px' weight={300} uppercase>Bitcoin</Text>
           <Text size='12px' weight={300}>
             {'@ '}
-            {quoteR
-              .map(q => {
-                if (q.baseCurrency.toLowerCase() === 'btc') return '$' + Currency.formatFiat((1 / (Math.abs(q.baseAmount / 1e8))) * Math.abs(q.quoteAmount))
-                else return '$' + Currency.formatFiat((1 / (Math.abs(q.quoteAmount / 1e8))) * Math.abs(q.baseAmount))
-              })
-              .getOrElse(
-                <Fragment>
+            {
+              Remote.Loading.is(quoteR)
```

Comment:
Because the SFOX API returns an error for quotes under the minimum, but does not for quotes over the maximum, the rate display had to be refactored and now we won't show a rate if the quote is under the min.